### DISCUSSION TITLE
php artisan make:form-field - uses App\Form insted of App\Filament\Form

### DISCUSSION THREAD

**Discussion by hrvoj3e** [filamentphp/filament#4336] [Help] [answered]
Is this a bug or is documentation old?
 - https://filamentphp.com/docs/2.x/forms/fields#building-custom-fields

```
filament/filament                      v2.16.19 Effortlessly build TALL-powered admin panels.
filament/forms                         v2.16.19 Effortlessly build TALL-powered forms.
```

Using 
```
php artisan make:form-field PlaceholderText
```
produces
```
app/Forms/Components/PlaceholderText.php
resources/views/forms/components/placeholder-text.blade.php
```
insted of
```
app/Filament/Forms/Components/PlaceholderText.php
resources/views/filament/forms/components/placeholder-text.blade.php
```

**Comment by danharrin** [comment, marked as answer]
that's correct :) all the forms & tables related stuff rarely has a `filament` prefix

**Reply by hrvoj3e** [reply]
So docs need updating then?

**Reply by danharrin** [reply]
i cant see where it's wrong?

**Reply by hrvoj3e** [reply]
Sorry. Tunnel vision. My bad. :confused: 
I looked at `use` statement in docs and expected a file in `App\Filament\Forms`.

But, it would be nice if the docs said 
> This will create the following field class (`app/Forms/Components/RangeSlider.php`) :
